app/models/rateitapp/rating.rb

```diff
@@ -19,5 +19,12 @@ class Rating < ActiveRecord::Base
     validates :ratable_type, presence: true
     validates :ratable_id, presence: true
     validates :user_id, presence: true
+    validate :value_is_within_range
+
+    def value_is_within_range
+      if (value && ratable_type) && !(0..5).cover?(value)
+        errors.add(:value, "can't be outside range")
+      end
+    end
```

Comment:
I forgot to ask you about this last week. I guess if a user hits reset or delete on a rating it would go to nil and not 0, correct? So in that case the limit would be 1..5